src/select.tsx

```diff
@@ -52,14 +52,18 @@ const useStyles = createUseStyles((theme) => ({
         left: 12 + 1, // 12 for select paddingLeft and +1 for border
         right: 48 + 1, // 48 for icon and +1 for border
         overflow: 'hidden',
-        top: ({label}) => (label ? 27 : 17),
+        top: ({label}) => (label ? 30 : 17),
```

Comment:
I don't understand why this label logic, I didn't see any changes if I change 17 value :/ @atabel. Maybe we can remove it?